csrc/python_frontend/fusion_definition.cpp

```diff
@@ -228,6 +228,14 @@ std::string FusionDefinition::fusionIr() {
   return ss.str();
 }
 
+std::string FusionDefinition::userScheduleIr() {
+  NVF_CHECK(id().has_value(), "Invalid fusion definition!");
+  NVF_CHECK(user_sched_ != nullptr, "UserSchedule is not valid!");
```

Comment:
Maybe we can relax the check here a bit. i.e. return an error message when user_schedule is not available, instead of an assert.